source-academy/frontend
# Fix alignment of logos for new login page for Firefox browsers

### Description

This PR primarily aims to improve the design and placement of the logos on the new login page to be more centered. Also address a small positioning problem with Firefox browsers on the login page.

I have tested the code to ensure no regressions in terms of logo display, and that it renders consistently across different screen resolutions and sizes.

### Type of change


### How to test

Ensure new login page displays logo centered on Chromium-based browsers, and for Firefox that the logo is no longer out of position.

### Checklist
